.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,44 @@
+name: Test Windows
+
+on:  [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: [python3]
+
```

Comment:
You could add matrix variables for the different msvc versions that you want to run with. I'm not sure how it works with faber, but with b2 you could put:

    cxxtoolset: [msvc-14.0, msvc-14.1, msvc-14.2]
    
Then at invocation time

    b2 toolset=${{ matrix.cxxtoolset }}

Reply:
Hang on, it I'm not sure that is valid with `windows-latest`. I was thinking I saw multiple visual studio versions installed on that image, but [it looks like there is only one](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#visual-studio-enterprise-2019). 

That one does have the toolsets for 14.0, 14.1 and 14.2 all installed, but I've never had luck building boost across the different toolsets from one Visual Studio install, I've always needed more dependencies from visual studio than just the toolset.

Reply:
The `msvc` module in faber will discover all installed versions, then instantiate a compiler for each, which can be selected on the command-line by specifying the version via `cxx.version=...` (see https://github.com/boostorg/python/blob/develop/.appveyor.yml#L89)